module.txt

```diff
@@ -5,10 +5,13 @@
     "displayName": "Climate Conditions",
     "description": "Module that introduces humidity and temperature fields and their modifiers",
     "dependencies": [
-            { "id": "AlterationEffects","minVersion": "1.0.0"},
-            { "id": "Health", "minVersion": "1.0.0" }
+                        { "id": "AlterationEffects","minVersion": "1.0.0"},
+                        { "id": "BiomesAPI", "minVersion": "4.0.0"},
+                        { "id": "Health", "minVersion": "1.0.0" },
+                        { "id": "Thirst", "minVersion": "1.0.0"}
         ],
```

Comment:
whilst your at this, could you do me the favor and fix the indentation here? :sweat_smile: